packages/async-flow/src/bijection.js

```diff
@@ -100,14 +100,14 @@ export const prepareBijection = zone => {
       const hostToGuest = h2g.for(self);
 
       if (guestToHost.has(g)) {
-        guestToHost.get(g) === h ||
+        toPassableCap(guestToHost.get(g)) === toPassableCap(h) ||
           Fail`internal: g->h ${g} -> ${h} vs ${guestToHost.get(g)}`;
         hostToGuest.get(h) === g ||
           Fail`internal h->g: ${h} -> ${g} vs ${hostToGuest.get(h)}`;
         return true;
       } else {
         !hostToGuest.has(h) ||
-          Fail`internal: unexpected h->g ${h} -> ${guestToHost.get(h)}`;
```

Comment:
Fixed another bug discovered while testing.

Reply:
Are the tests that found these going to be included?

Reply:
Yes, they are now.